tests/test_n26_importer.py

```diff
@@ -109,11 +109,14 @@ def test_extract_no_transactions(importer, filename):
 
 
 def assert_transaction(
-    transaction: Transaction,
-    date: datetime,
+    transaction: Transaction,  # type: ignore
+    date: datetime.date,
     payee: str,
     narration: str,
-    postings: List[Tuple[str, Optional[str], Optional[Decimal]]],
+    postings: List[
+        Tuple[str, Optional[str], Optional[Decimal]]
+        | Tuple[str, str, Decimal, Tuple[str, Decimal]]
+    ],
```

Comment:
I wonder if you could directly use `beancount.core.data.Posting` in the type annotation here to make `pytest` stop complaining, such that the final annotation is `List[Posting]`.